app/src/main/assets/About.html

```diff
@@ -8,6 +8,7 @@ <h4>Designed and developed by</h4>
   <li>Thomas Arnold</li>
   <li>Christoph K&ouml;nig</li>
   <li>Torbjörn Eklund</li>
+  <li>kuhrusty</li>

```

Comment:
@kuhrusty thanks again for the support!  Open Beta will be out soon to test more.